packages/components/breadcrumb/src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0"
+        />
+        <title>Stencil Component Starter</title>
```

Comment:
maybe you need to change the title

Reply:
PR is still a draft, some cleaning up to do ;)